contracts/governance/TrueFiVault.sol

```diff
@@ -38,6 +36,7 @@ contract TrueFiVault is Initializable {
 
     function initialize(
         address _beneficiary,
+        address _finalOwner,
```

Comment:
would be unfortunate if we provide both wrong beneficiary and final owner, but ok, we'll just have to double check

Reply:
yes, we will just double check. If we make a mistake we have to upgrade the smart contract, this is intentional

Reply:
It looks like `owner` is no longer used anywhere in this contract. Can we remove it?

If we want to support future `owner` authorization, perhaps we should inherit UpgradeableClaimable instead of Initializable. That seems to be our canonical way of expressing contract ownership.